app/scripts/modules/core/src/entityTag/EntityTagEditor.tsx

```diff
@@ -73,39 +69,30 @@ export class EntityTagEditor extends React.Component<IEntityTagEditorProps, IEnt
   constructor(props: IEntityTagEditorProps) {
     super(props);
 
-    const { tag } = this.props;
-    const ownerIndex = this.props.ownerOptions ? 0 : -1; // Assuming that the first option is the provided option
+    const { tag, ownerOptions: opts } = this.props;
+    const ownerOptions = opts || [];
+    const defaultOwnerOption = (ownerOptions || []).find(opt => opt.isDefault) || ownerOptions[0];
```

Comment:
redundant `|| []`

Reply:
Looks like this is the only remaining actionalble thing...
```suggestion
    const defaultOwnerOption = ownerOptions.find(opt => opt.isDefault) || ownerOptions[0];
```